### PR TITLE
increase width of buttons to full screen on attendees page

### LINKED ISSUE #181: Fix the column width issue in tickets/attendees
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Screenshots : 
Current : 
attendees

Expected : 
The buttons should occupy whole width of screen.

### PR DESCRIPTION
**Pull request by sumedh123**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #181 

Screenshots : 
Fix : 
attendees

### CODE REVIEW

File: app/templates/events/view/tickets/attendees.hbs [hunk 1]
```diff
@@ -46,7 +46,7 @@
   </div>
   <div class="row">
     <div class="ui hidden divider"></div>
-    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+    <div class="ui {{if device.isMobile 'eight wide column vertical'}} basic buttons">
```
Comment (niranjan94) [inline app/templates/events/view/tickets/attendees.hbs]: Instead of `eight wide column` try `fluid`

**Comment by CosmicCoder96** [comment]
@niranjan94 Please see the discussion on the issue ( #181 )  once, so should this PR include the fix for #124 because #181 is basically not required if #124 is solved, thank you!

**Comment by geekyd** [comment]
@niranjan94 This is not required please check the  #124, if we solve it we do not need to solve #124. Please guide.

**Comment by niranjan94** [comment]
True @geekyd and @CosmicCoder96 . @sumedh123 I'm closing this PR and #181. Please help with #124 and propose a solution that could be used throughout the app.

**Pull request closed without merging**